tasks/cleanup-workspace/cleanup-workspace.yaml

```diff
@@ -29,6 +33,12 @@ spec:
         #!/usr/bin/env sh
         set -eux
 
+        if [ ! -z "$(params.pipelineRunUid)" ] ; then
```

Comment:
I suggest to use `-n` instead of `! -z`. I believe they are equivalent.

Reply:
Updated